graphql-server/src/schemas/databricks.ts

```diff
@@ -1,101 +0,0 @@
-import { AuthenticationError, ForbiddenError } from "apollo-server-express";
```

Comment:
just to confirm - all of this is either refactored into patient.ts and/or moved to the utils/databricks ?